api/handlers/zip_package.go

```diff
@@ -37,6 +37,20 @@ func (m *ZippedPackage) GetResource(resourceName string) (io.ReadCloser, error)
 	return file, nil
 }
 
+func (m *ZippedPackage) CheckIfResourceExists(resourceName string) error {
+	actualPath := path.Clean(path.Join(m.extractedDir, resourceName))
+	if !strings.HasPrefix(actualPath, m.extractedDir) {
+		return ErrorInvalidResourcePath
+	}
+
+	_, err := os.Stat(actualPath)
```

Comment:
maybe we should check a few more things (like that it's not a directory for example)